platform/ios/src/MGLScaleBarView.h

```diff
@@ -0,0 +1,5 @@
+#import <UIKit/UIKit.h>
+
+@interface MGLScaleBarView : UIView
+
```

Comment:
I think knobs along these lines could be tail work. That way we can address customizability of the ornaments at the same time in a consistent manner.